resources/packs/github/github_repo.go

```diff
@@ -448,29 +448,36 @@ func (g *mqlGithubBranch) GetProtectionRules() (interface{}, error) {
 
 	var ghDismissalRestrictions *githubDismissalRestrictions
 
-	if branchProtection.RequiredPullRequestReviews.DismissalRestrictions != nil {
-		ghDismissalRestrictions = &githubDismissalRestrictions{
-			Users: []string{},
-			Teams: []string{},
-		}
+	rprr, err := core.JsonToDict(githubRequiredPullRequestReviews{})
```

Comment:
why do you not initialize this as `nil`? It's making the code shorter and it won't do unnecessary work of serializing stuff.
```suggestion
	var rprr map[string]interface{}
```

Reply:
Good point. Thanks.